### DISCUSSION TITLE
classical pressure

### DISCUSSION THREAD

**Discussion by matukumilli** [SSCHAcode/python-sscha#75] [Q&A] [answered]
Dear SSCHA developers,

Is there a way to exclude zero-point energy contribution in the SSCHA dynamical matrices? In order to understand the role of zero-point motions, phonon frequencies with and without zero-point energy are required. As it has been described as "classical pressure" in Fig.4 of Monacelli et al., PRB,2018 [DOI:10.1103/PhysRevB.98.024106], I believe it is possible with SSCHA code. 

I request developers' suggestion on this.

Regards,
Prasad Matukumilli
Post Doc | TSU | JNCASR

**Comment by mesonepigreco** [comment, marked as answer]
Hi, sorry for the delay in the answer. 

The best way to currently do this is to hack a bit the system, i.e. multiplying the masses for a large factor (like 10000).
In this way, your calculation becomes classical and quantum effects becomes negligible.  

Then on the final dynamical matrices you can restore the original masses to get the correct frequencies. 

We are planning to add a feature in the code to switch on and off quantum effects, but still is not available and this hack is the simplest way.


To modify the masses, you can simply the masses in Ry (adding 4 zeros on the tail before the dot for example) written at the beginning the first dynamical matrix.

**Reply by matukumilli** [reply]
This "atom to planet" trick should be followed through all populations in a minimization run? or just last for last population would possibly evade quantum effects?

Regards,
Prasad

**Reply by mesonepigreco** [reply]
Hi. 
You need to extract a new ensemble, right now is still not possible to use the reweighting when masses change and thus recycle the same ensemble extracted with the quantum dynamical matrix.
But, indeed, it is convenient to start with the final dynamical matrix you obtained with a standard SSCHA calculation, and, hopefully, you converge in just one iteration.
